src/lib/containers/home_page/people/UserCardListGroups.Desktop.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react'
+import UserCardListRotate from '../../../containers/UserCardListRotate'
+import { FacetColumnValuesRequest } from '../../../utils/synapseTypes'
+import { UserCardListGroupsProps } from './UserCardListGroups'
+
+
+export default function UserCardListGroupsDesktop(props: UserCardListGroupsProps) {
+  const [selectedGroupIndex, setSelectedGroupIndex] = useState<number>(0)
```

Comment:
The desktop version is a bit optimized, it will only run the query for the menu item selected.  It just so happens that the menu items correspond to the facet values that I want to display in this use case.  If this is not true in another case, we'll need another parameter (to indicate the menu item name in addition to the facet value).